software/spack_env/summit_spack_env.rst

```diff
@@ -0,0 +1,126 @@
+Summit
+############################################
+
+This example is intended for use on Summit with RHEL 7 as the OS.
+
+Purpose
+--------------------------------------------
+
+This guide meant as an example for a user to setup a Spack environment for application development using the OLCF 
+provided files as a template.
+
+.. warning::
+
+  The provided ``spack.yaml`` files are templates for a user to use as an example.
+
+  This not intended as a guide for a new Spack user.  Please see `the Spack 101 tutorial <https://spack-tutorial.readthedocs.io/en/latest>`_
+  if you need assistance starting out with Spack.
+
+The provided Spack environment files are intended to assist OLCF users in setup their development environment at the 
+OLCF.  The base environment file includes the compilers and packages that are installed at the system level.
+
+Traditionally, the user environment is modified by module files.  For example, a user would add use  ``module load cmake/3.18.2`` to 
+load CMake version 3.18.2 into their environment.  Using a Spack environment, a user can add an OLCF provided package
+and build against it using Spack without having to load the module file separately.
+
+The information presented here is a subset of what can be found at `the Spack documentation site <https://spack.readthedocs.io/>`_.
+
+.. note:: 
+  Definitions
+
+  - Spack environment - A set of Spack specs for the purpose of building, rebuilding and deploying in a coherent fashion.
+
+  - External Packages - An externally-installed package used by Spack, rather than building its own package.
+
+Getting Started
+--------------------------------------------
+
+Clone the OLCF User Environment repo and the Spack repo, start a new Spack instance, and create and activate a new Spack environment:
+
+.. code-block:: none
+
+  ## From a Summit login
+
+  > git clone https://code.ornl.gov/2ff/olcf-user-environments.git
+  > cd olcf-user-environments
+
+  > git clone https://github.com/spack/spack.git
+  > source spack/share/spack/setup-env.sh
+
+  > spack env create my_env linux-rhel7-ppc64le/summit/spack.yaml
+  > spack env activate my_env
+
+Now a user can add and install their dependencies with Spack and proceed with developing their application.
+
+Add Dependencies to the environment
+--------------------------------------------
+
+Adding OLCF Installed Dependencies as External Packages
+-------------------------------------------------------
+
+If an OLCF installed package is available, these can be added via the ``spack.yaml`` file by adding to the ``packages``
```

Comment:
Can you specify that the spack.yaml in this case is the spack.yaml file for the environment you created (for example in spack/var/spack/environments/my_env/spack.yaml) and not the original spack.yaml we used to create the env in the first place. That's what tripped me up last time.

Reply:
updated with commit